cli/kubectl_kadalu/main.py

```diff
@@ -1,12 +1,15 @@
+"""
+This is an CLI command to handle kadalu executions
+"""
 from argparse import ArgumentParser
 
-from kubectl_kadalu.storage_add import storage_add_args, subcmd_storage_add, \
-    storage_add_validation
+from kubectl_kadalu.storage_add import storage_add_args
```

Comment:
Removed methods used in Line 30-31. May be we need to skip unused imports

Reply:
Good catch! Fixed.